mne/stats/regression.py

```diff
@@ -32,28 +32,30 @@ def linear_regression(inst, design_matrix, names=None):
         the first dimension of the data. The first column of this matrix will
         typically consist of ones (intercept column).
     names : array-like | None
-        Optional parameter to name the regressors. If provided, the length must
-        correspond to the number of columns present in regressors
-        (including the intercept, if present).
-        Otherwise the default names are x0, x1, x2...xn for n regressors.
+        Optional parameter to name the regressors (i.e., the columns in the
+        design matrix). If provided, the length must correspond to the number
+        of columns present in design matrix (including the intercept, if
+        present). Otherwise the default names are ``'x0'``, ``'x1'``,
+        ``'x2', …, 'xn'`` for ``n`` regressors.
```

Comment:
chiming in too late here since you already merged, but this is not correct.  If there are `n` regressors and you start at `x0` then your last regressor will be `x(n-1)`.

Reply:
good point. I actually didn't check this for logical correctness, simply adapted to formatting of the existing text. Will fix.

Reply:
This has been fixed via #9239